tracing-subscriber/src/fmt/format.rs

```diff
@@ -141,6 +157,19 @@ impl<F, T> Format<F, T> {
         }
     }
 
+    /// Use the full JSON format.
+    ///
+    /// See [`Json`].
+    #[cfg(feature = "json")]
+    pub fn json(self) -> Format<Json, T> {
+        Format {
+            format: PhantomData,
+            timer: self.timer,
+            ansi: self.ansi,
```

Comment:
I don't this one is ever used for JSON output. Should we disable this for JSON, or should we add documentation on the builder to note that its disabled for JSON?

Reply:
It kind of does nothing in here. I think we should disable it though...

Reply:
How about both, to avoid surprises?

Reply:
Btw. Asking a stupid question, but do we test separately with different feature flags? Like with and without JSON? If I start doing cfg attributes such as `#[cfg(all(feature = "ansi", not(feature = "json")))]` this might lead to some tests not running, if always tested with all flags.

Reply:
A bit like that, but I wouldn't try to test with specific feature flag combinations. We'll probably move this to a separate module in the coming future, so I'm okay the `"json"` flag being coupled with the `"ansi"` flag.

Reply:
In this way maybe... Now we can't use ANSI at all of enabling json feature. https://github.com/pimeys/tracing/commit/3e4c8d9a10862b8770e2dc0f36b5bb1d46d0e394

Reply:
Note https://github.com/pimeys/tracing/commit/9d4f598637d4ddfdd3be3f97702b388bf51e4a53